### PR TITLE
Update versions, fixing producer for dojot.tenancy

### PR DESCRIPTION
**Pull request by giovannicuriel**
* **Please check if the PR fulfills these requirements**
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
This PR updates the dojot.module version to 0.0.1a4, which has important bug fixes.

* **What is the current behavior?** (You can also link to an open issue here)
Bugs.

* **What is the new behavior (if this is a feature change)?**
No bugs, or at least fewer bugs.

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
No.

* **Is there any issue related to this PR in other repository?** (such as dojot/dojot)
No.

* **Other information**:

### CODE REVIEW

File: auth/controller/KafkaPublisher.py [hunk 1]
```diff
@@ -46,8 +46,8 @@ def init(cls):
                     "dr_cb": True
                 },
                 "consumer": {
-                    "group.id": "dojot.auth",
-                    "metadata.broker.list": kafka_host
+                    "group_id": "dojot.auth",
```
Comment (raulnegreiros) [inline auth/controller/KafkaPublisher.py]: why it is not configurable? Other services have the option to set the group_id, isn't?
In the dojot/docker-compose#96 PR was added the environment variable KAFKA_GROUP_ID, but where it is used?

File: auth/controller/KafkaPublisher.py [hunk 2]
```diff
@@ -64,11 +64,16 @@ def init(cls):
 
         LOGGER.debug("Config is:  " + json.dumps(config.auth))
         cls.messenger = Messenger("dojot.auth", config)
+        LOGGER.debug(f"Creating faux-tenant {dojot_management_tenant}...")
```
[the remark this answers is not in the document]
Comment (giovannicuriel) [reply auth/controller/KafkaPublisher.py]: Like false tenant, but in a fancy way. As in faux leather.

File: auth/controller/KafkaPublisher.py (hunk 1 above)
Comment (giovannicuriel) [reply auth/controller/KafkaPublisher.py]: I'll remove this part. If Kafka configuration is not set, the library will set it to its default value.

File: auth/controller/KafkaPublisher.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (ericbaum) [reply auth/controller/KafkaPublisher.py]: You, sir, are very fancy

**Pull request merged**